### PR TITLE
content: Document templateMetrics usage

### LINKED ISSUE #213: Document templateMetrics usage
Need to document `--templateMetrics` (https://github.com/gohugoio/hugo/commit/b4a14c25fe85c41b79497be27ead128502a4dd7b).

### PR DESCRIPTION
**Pull request by moorereason**
Fixes #213

### CODE REVIEW

File: content/commands/hugo.md [hunk 1]
```diff
@@ -67,7 +67,6 @@ hugo [flags]
 
 ### SEE ALSO
 * [hugo benchmark](/commands/hugo_benchmark/)	 - Benchmark Hugo by building a site a number of times.
-* [hugo check](/commands/hugo_check/)	 - Contains some verification checks
```
Comment (bep) [inline content/commands/hugo.md]: THis file is autogenerated and was correct.

Comment (moorereason) [reply content/commands/hugo.md]: I committed what `hugo gen doc` generated.

**Comment by digitalcraftsman** [comment]
@moorereason could you also add a few lines that describe the given metrics, namely the columns of the output?

To quote @nraboy from the [forum](https://discourse.gohugo.io/t/performance-expectations/8541):

> That is a lot of information, most of which I am not sure how to decipher.

**Comment by moorereason** [comment]
@digitalcraftsman, can the column descriptions be in the Build Performance page from #216?

**Comment by digitalcraftsman** [comment]
That should be fine but people should have a link to the page in #216. Otherwise it's unlikely that users will find it without using the search.

As the pages in this PR are autogenerated is it possible to add a link the see also page?

**Comment by digitalcraftsman** [comment]
What's the status of this commit (i.e. see Bjørn Erik's comment above)?

**Comment by moorereason** [comment]
It's ready to commit as far as I'm concerned.

**Pull request merged**